components/ListView.js

```diff
@@ -67,8 +67,10 @@ class ListView extends React.Component {
   static propTypes = {
     autoHideHeader: React.PropTypes.bool,
     style: React.PropTypes.object,
-    data: React.PropTypes.array,
+    data: React.PropTypes.oneOfType([
+      React.PropTypes.object, React.PropTypes.array]),
```

Comment:
When would you use object over array? Could you write an example of passed data as object and what would you expect to be rendered?